issue_checker/readme.md

```diff
@@ -0,0 +1,10 @@
+# Issue Checker
+
+The issue checker takes a composer.json file and checks the issue status of any 
+patches that are applied in it.
+
+## Usage
+
+```
+/path/to/issue_checker/index.js /path/to/composer.json
```

Comment:
@Kingdutch the composer.json was not needed right? Can you update it in the readme?